### ISSUE TITLE
Let's fail more silently when a wrong app-id is provided

### ISSUE THREAD

**Issue by michael** [michael/hub#9]
```
Composer application f7043dc691102f3ac3175e606af2c8cb with secret ca85e9a193c721e5d65eba26164c0d87
NoRecordFound: No application found with id f7043dc691102f3ac3175e606af2c8cbX
    at db.one [as callback] (/Users/michael/projects/hub/lib/db.js:57:26)
    at p.handleReadyForQuery (/Users/michael/projects/hub/node_modules/pg/lib/query.js:79:10)
```

**Comment by yuchi** [comment]
Mh... why is it logging the error on startup?

**Comment by michael** [comment]
No on startup it's fine. This is being logged when a new authorization is being created but there's a non-existent client_id provided.

`POST /api/v1/authorizations`

**Comment by yuchi** [comment]
Argh. Error handling still needs some love

**Comment by yuchi** [comment]
Solved by #10 

ATM returns the following:

``` json
{
  "name": "NoRecordFound",
  "message": "No application found with id 10ab06a48395379bc8a50ed3a8d9dd4"
}
```
